site/public/js/admin-team-form.js

```diff
@@ -166,6 +176,7 @@ function adminTeamForm(new_team, who_id, reg_section, rot_section, user_assignme
                 break;
             }
         }
+        // eslint-disable-next-line eqeqeq
         if (found || $(this).val() == '') {
```

Comment:
We could replace this with `===` only if we're confident that we're not relying on entering this if statement when `$(this).val()` is `0` or `[]` as those are both `== ''`